CMakeLists.txt

```diff
@@ -301,13 +308,13 @@ set(Boost_USE_STATIC_RUNTIME OFF)
 set(Boost_ADDITIONALVERSION "1.51" "1.51.0")
 find_package(Boost COMPONENTS system filesystem regex REQUIRED)
 
+set(GLEW_USE_STATIC_LIBS ${GLEW_STATIC})
 find_package(GLEW REQUIRED)
 
 if (OPENAL_SOUND)
     find_package(OpenAL REQUIRED)
-    include_directories(${OPENAL_INCLUDE_DIR})
-    find_package(LibSndFile REQUIRED)
 endif()
+find_package(SndFile REQUIRED)
```

Comment:
This is not required if sound support is disabled, as it's not used anywhere in that case

Reply:
It is still required though or I wouldn't have moved this :) I don't know the sound part of the code but if this library shouldn't be required if there's no sound then the abstract interfaces should be rewritten. I had 4 linker errors IIRC so there are at least 4 places where we use libsndfile regardless if OPENAL_SOUND is true or not.

Reply:
That's weird and sounds like a bug. I remember we used to have some problems with OpenAL on MXE builds and back then it built fine with OPENAL_SOUND set to OFF.

Reply:
So should I try to fix this in this PR or leave it for another PR/someone else to fix?

Reply:
I think you can just leave it for now